test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -146,4 +176,9 @@ public static boolean isAwsExecution() {
         return dockerConfig.trim().equalsIgnoreCase("true") ?  true : false;
 
     }
+
+    public static boolean isSecurityEnabled() {
```

Comment:
Make the method more specific like `isAuthEnabled()`, etc.

Reply:
Same method would be used once we have Auth+TLS environment for security system tests.
Does not make sense to have different methods for AuthEnabled and TLSEnabled unless we plan to have different environment setups with only auth enabled or only tls enabled.

Reply:
To make this intent clear, it might be a good idea to create more specific methods like `isAuthEnabled` and then use a wrapper method like `isSecurityEnabled()` that brings them together for convenience and making the intention clear. That being said, I'll leave it up to you.

Reply:
done